### ISSUE TITLE
[TAG] Use USVString for `moduleURL` in audioWorklet.import()

### ISSUE THREAD

**Issue by hoch** [WebAudio/web-audio-api#965]
The definition of import in the [Worklet Spec] is:

```
interface Worklet {
    [NewObject] Promise<void> import(USVString moduleURL);
};
```

whereas the interface in WebAudio uses DOMString:

```
interface AudioWorklet : Worklet {
    Promise<void> import (DOMString moduleUrl);
};
```

The issued is raised by @slightlyoff at #869.

**Comment by hoch** [comment]
In the PR under the review, I removed the AudioWorklet interface because of its redundancy.

https://github.com/WebAudio/web-audio-api/pull/963/files#diff-eacf331f0ffc35d4b482f1d15a887d3bL5155

**Comment by hoch** [comment]
Fixed by #963. The `audioWorklet` becomes just a `Worklet`.
